### PR TITLE
IDVA6-937 Pagination for Manage authorised people page

### PR DESCRIPTION
**Pull request by bmchugh-ch**
Link to Jira:
https://companieshouse.atlassian.net/browse/IDVA6-937

Changes:

Adds pagination component to the manage authorised people page

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: test/src/lib/helpers/buildPaginationHelper.test.ts [hunk 1]
```diff
@@ -213,3 +213,20 @@ describe("setLangForPagination", () => {
         expect(() => setLangForPagination(pagination, translations)).not.toThrow();
     });
 });
+
+describe("stringToPositiveInteger should convert page query param to a positive integer, 1 or greater", () => {
+    it.each([
```
Comment (tferns-ch) [inline test/src/lib/helpers/buildPaginationHelper.test.ts:218]: love these parameterized tests

**Review by tferns-ch — DISMISSED** [review]
nice one, cant see anything to suggest!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by mmodzelewski-ch — APPROVED** [review]

**Pull request merged**